netbout-web/pom.xml

```diff
@@ -326,13 +326,15 @@
                             <npmVersion>2.11.3</npmVersion>
                         </configuration>
                     </execution>
+                    <!-- @todo #914 investigate error messages of the following
+                         command and change package.json accordingly -->
```

Comment:
@jhyle can you add more details here like what exactly errors do we get? Puzzle description can be long, no worries.